### ISSUE TITLE
V1.1.16: warning when clear queue

### ISSUE THREAD

**Issue by erkr** [music-assistant/hass-music-assistant#246]
Having a paused queue, or a completed queue.
Then clearing all items in the queue, this warning is logged:
```
Logger: pychromecast.controllers
Source: /usr/local/lib/python3.9/site-packages/pychromecast/controllers/media.py:596 
First occurred: 17:09:36 (2 occurrences) 
Last logged: 18:11:21

STOP command requested but no session is active.
```
Not harmful, but should not occur either. Not seen earlier.

**Comment by marcelveldt** [comment]
It isn't harmful and thrown by cast integration when a stop command is sent when no session is active. If you send the "media_stop" command yourself to a player you will receive this same warning.

A stop command is sent just in case on some occasions, such as clearing the queue and power off.
I'll see if I can add another check there but if I remember correctly that is not always reliable in case of buffering etc.

**Comment by marcelveldt** [comment]
Fix released in 1.1.17

**Comment by erkr** [comment]
Will test today. Thanks!
